runtime-decompiler/src/main/java/org/jrd/frontend/MainFrame/BytecodeDecompilerView.java

```diff
@@ -80,21 +80,20 @@ public void changedUpdate(DocumentEvent documentEvent) {
             @Override
             public void mouseClicked(MouseEvent e) {
                 final String name = filteredClassesJlist.getSelectedValue();
-                new SwingWorker<Void, Void>() {
```

Comment:
btw title of this message is a bit of missleading

Reply:
Message?

Reply:
yes, commit message